src/js/apps/clinicians/sidebar/clinician-sidebar_app.js

```diff
@@ -26,6 +26,5 @@ export default App.extend({
     this.clinician.trigger('editing', false);
 
     Radio.request('sidebar', 'close');
-    Radio.trigger('event-router', 'clinicians:all');
```

Comment:
We'll need to keep this line.

And somehow have it only run when the sidebar is being closed via the `X` icon button. Not when routing to the worklist (or elsewhere).